AndroidAnnotations/androidannotations-core/androidannotations-test/src/test/java/org/androidannotations/test/efragment/MyListFragmentTest.java

```diff
@@ -123,6 +127,40 @@ public void ignoredWhenDetachedWorksForIgnoredMethod() {
 	}
 
 	@Test
+	public void notIgnoredAfterOnCreateView() {
+		assertFalse(myListFragment.didExecute);
+		LayoutInflater inflater = mock(LayoutInflater.class);
+		ViewGroup viewGroup = mock(ViewGroup.class);
+		Bundle bundle = mock(Bundle.class);
+		myListFragment.onCreateView(inflater, viewGroup, bundle);
```

Comment:
We could just pass `null` to every parameter here, there is no need for mocking in this case.